sorc/global_cycle.fd/machine.f90

```diff
@@ -8,17 +8,8 @@ MODULE MACHINE
 
  IMPLICIT NONE
  SAVE
-!  Machine dependant constants
- integer, parameter :: kind_io4  = 4
- integer, parameter :: kind_io8  = 8
- integer, parameter :: kind_ior = 8
- integer, parameter :: kind_evod = 8
- integer, parameter :: kind_dbl_prec = 8
- integer, parameter :: kind_rad  = selected_real_kind(13,60) !< the '60' maps to 64-bit real
- integer, parameter :: kind_phys = selected_real_kind(13,60) !< the '60' maps to 64-bit real
- integer, parameter :: kind_REAL = 8 !< used in cmp_comm
- integer, parameter :: kind_INTEGER = 4 !< -,,-
-!
- real(kind=kind_evod), parameter :: mprec = 1.e-12 !< machine precision to restrict dep
-
+ integer, parameter :: kind_io4  = 4 !< Kind type for 4-byte float point
+                                     !! variables.
+ integer, parameter :: kind_io8  = 8 !< Kind type for 8-byte float point
+                                     !! variables.
```

Comment:
You removed a bunch of integers - is that intentional? If so, please enter an issue describing this code change so we have a record.

Reply:
See #412